Get specific user

JethroWarnett opened the pull request.

JethroWarnett committed:
Update db.sqlite

JethroWarnett committed:
Added functionality to get specific user

Logged In User is able to access his/her attributes

dominiqueroux commented:
Works, (in the next version: Only an admin should be allowed to see all users)
=> Can be merged as soon as db problem is fixed

JethroWarnett committed:
Revert "Added functionality to get specific user"

This reverts commit 5621997d2e9c650c5cb7df39f450261616b37312.

JethroWarnett committed:
Revert "Revert "Added functionality to get specific user""

This reverts commit 142a4f722c2d4fb44b749d14044a0928da8a7735.

JethroWarnett committed:
Revert "Update db.sqlite"

This reverts commit dad919bba446317b0ba2df77e46559d2d719ae95.

JethroWarnett committed:
Only Admin can get all Users

Fixed the code, so that the method getAll can only be accessed by the Admin. Only changed code in the method getAll

Commit:
reset the database to the master version

dominiqueroux committed:
Merge branch 'master' into GetSpecificUser
